x-pack/plugins/encrypted_saved_objects/server/mocks.ts

```diff
@@ -5,24 +5,25 @@
  */
 
 import { EncryptedSavedObjectsPluginSetup, EncryptedSavedObjectsPluginStart } from './plugin';
-import { EncryptedSavedObjectsClient } from './saved_objects';
+import { EncryptedSavedObjectsClient, EncryptedSavedObjectsClientOptions } from './saved_objects';
 
 function createEncryptedSavedObjectsSetupMock() {
   return {
     registerType: jest.fn(),
     __legacyCompat: { registerLegacyAPI: jest.fn() },
     usingEphemeralEncryptionKey: true,
+    startWithHiddenTypes: jest.fn(),
```

Comment:
__nit:__ looks like we don't need this method?

Reply:
huh looks like a merge issue, good catch